packages/kit-bg/src/services/ServiceFirmwareUpdate/ServiceFirmwareUpdate.ts

```
@@ -349,6 +349,11 @@ class ServiceFirmwareUpdate extends ServiceBase {
     await firmwareUpdateRetryAtom.set(undefined);
     serviceHardwareUtils.hardwareLog('checkAllFirmwareRelease');
 
+    const currentTransportType =
+      await this.backgroundApi.serviceSetting.getHardwareTransportType();
+    const isBleTransport =
+      currentTransportType === EHardwareTransportType.BLE;
```

ESLint

[error] Delete `⏎·····`

(prettier/prettier)